defaults/main.yml

```
@@ -14,9 +14,9 @@ gitlab_gpg_key_id: "F6403F6544A38863DAA0B6E03F01618A51312F3F"
 
 gitlab_gpg_old_key_ids: []
 
-gitlab_runner_docker_machine_binary_url: "https://gitlab.com/gitlab-org/ci-cd/docker-machine/-/releases/v0.16.2-gitlab.15/downloads/docker-machine-Linux-{{ ansible_architecture }}"
+gitlab_runner_docker_machine_binary_url: "https://gitlab.com/gitlab-org/ci-cd/docker-machine/-/releases/v0.16.2-gitlab.20/downloads/docker-machine-Linux-{{ ansible_architecture }}"
 
-gitlab_runner_docker_machine_binary_checksum: "sha256:dc92e2d2a293d66545eb0719d4816d5ebc7ac9ba5495823bbf4eb01c6da37a6e"
+gitlab_runner_docker_machine_binary_checksum: "sha256:4eb1bdf119496d35fd6f61be8235343919563ef16865aace390e6837aafee25a"
```

I wonder if it also works if we specify the file instead:
```suggestion
gitlab_runner_docker_machine_binary_checksum: "sha256:https://gitlab.com/gitlab-org/ci-cd/docker-machine/-/releases/v0.16.2-gitlab.20/downloads/release.sha256"
```

---

Unfortunately it did not work. Would have been too good.

See https://github.com/hifis-net/ansible-role-gitlab-runner/actions/runs/4560608720/jobs/8045720269#step:5:260

---

This looks like a bug in the `get_url` module.